pom.xml

```diff
@@ -225,6 +225,11 @@
             <name>Mark Waite</name>
             <email>mark.earl.waite@gmail.com</email>
         </developer>
+        <developer>
```

Comment:
At the time I was using this plugin at work and intended to become an active maintainer, but since submitting this I have moved to a new job where I'm not using this plugin, and so I probably won't be contributing much here going forward.  I'll update the branch to remove this POM change.